nova/cmd/shellinaboxproxy.py

```diff
@@ -37,8 +37,9 @@ def main():
 
     config.parse_args(sys.argv)
     # Run mitmproxy with shellinaboxproxy.py as an inline script
-    check_output("mitmdump -R %s --port %s --bind-address %s --script %s" % (
-                 CONF.shellinabox.proxyclient_url,
-                 CONF.shellinabox.port,
-                 CONF.shellinabox.host,
-                 path.abspath(script)), shell=True)
+    cmd = ['mitmdump',
+           '--mode', f"reverse:{CONF.shellinabox.proxyclient_url}",
```

Comment:
Is there a known default port? From the helm-charts it seems like it's [only the FQDN](https://github.com/sapcc/helm-charts/blob/40840a93aa8f4bdebe6810d7ccad8e2876349415/openstack/utils/templates/_hosts.tpl#L137) of the host.

In `mitmdump --help` it says
```
SPEC is host specification in the form of "http[s]://host[:port]"
```

Reply:
The known default port for https is 443. We're not going to put that in.

Looking at a running pod, `mitmdump` is started with a full URL starting with `https://`.